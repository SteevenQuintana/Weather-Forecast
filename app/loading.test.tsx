import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import Loading from './loading'

describe('loading', () => {
  it('should match snapshot', () => {
    const { container } = render(<Loading />)
    expect(container).toMatchSnapshot()
  })
})
