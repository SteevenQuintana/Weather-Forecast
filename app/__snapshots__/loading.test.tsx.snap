// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loading should match snapshot 1`] = `
<div>
  <div
    class="flex flex-col sm:grid sm:grid-cols-[40%_60%] gap-4 min-h-[500px]"
    data-testid="loading"
  >
    <article
      class="h-full bg-[#3054b7ca]  rounded-2xl dark:bg-[#3a4957cc] mx-4 md:mx-0"
    />
    <article
      class="h-full bg-[#3054b7ca]  rounded-2xl dark:bg-[#3a4957cc] mx-4 md:mx-0"
    />
  </div>
</div>
`;
